test/e2e/data/infrastructure-docker/cluster-template.yaml

```diff
@@ -122,4 +122,39 @@ spec:
   unhealthyConditions:
     - type: E2ENodeUnhealthy
       status: "True"
-      timeout: 30s
\ No newline at end of file
+      timeout: 30s
+---
+apiVersion: addons.cluster.x-k8s.io/v1alpha3
+kind: ClusterResourceSet
+metadata:
+  name:  "${CLUSTER_NAME}-crs-0"
+spec:
+  clusterSelector:
+    matchLabels:
+      label: e2e
+  resources:
+    - name: test-secret1
+      kind: Secret
+    - name: test-configmap1
```

Comment:
Uhmm, I'm not sure I want an explicit error in cluster template (assuming config map and test-not exists).
Those type of tests are already covered by unit tests IMO, and if we want to have additional E2E test coverage I would recommend to use a different cluster template given that this one is used as example by developers

Reply:
Makes sense, I will create a new template.